model_compression_toolkit/quantizers_infrastructure/keras/inferable_quantizers/base_symmetric_inferable_quantizer.py

```diff
@@ -16,51 +16,58 @@
 
 import numpy as np
 
+from model_compression_toolkit.core.common.constants import FOUND_TF
 from model_compression_toolkit.quantizers_infrastructure.common.base_inferable_quantizer import QuantizationTarget
-from model_compression_toolkit.quantizers_infrastructure.keras.inferable_quantizers.base_keras_inferable_quantizer \
-    import \
-    BaseKerasInferableQuantizer
-from model_compression_toolkit.quantizers_infrastructure.keras.inferable_quantizers.base_uniform_inferable_quantizer \
-    import \
-    BaseUniformInferableQuantizer
 
 
-class BaseSymmetricInferableQuantizer(BaseUniformInferableQuantizer):
+if FOUND_TF:
+    from model_compression_toolkit.quantizers_infrastructure.keras.inferable_quantizers\
+        .base_uniform_inferable_quantizer \
+        import \
+        BaseUniformInferableQuantizer
 
-    def __init__(self,
-                 num_bits: int,
-                 threshold: np.ndarray,
-                 signed: bool,
-                 quantization_target: QuantizationTarget):
-        """
-        Initialize the quantizer with the specified parameters.
+    class BaseSymmetricInferableQuantizer(BaseUniformInferableQuantizer):
 
-        Args:
-            num_bits: number of bits to use for quantization
-            threshold: threshold for quantizing weights
-            signed: whether or not to use signed quantization
-            quantization_target: An enum which selects the quantizer tensor type: activation or weights.
-        """
+        def __init__(self,
+                     num_bits: int,
+                     threshold: np.ndarray,
+                     signed: bool,
+                     quantization_target: QuantizationTarget):
+            """
+            Initialize the quantizer with the specified parameters.
 
-        if not isinstance(threshold, np.ndarray):
-            threshold = np.asarray(threshold)
+            Args:
+                num_bits: number of bits to use for quantization
+                threshold: threshold for quantizing weights
+                signed: whether or not to use signed quantization
+                quantization_target: An enum which selects the quantizer tensor type: activation or weights.
+            """
 
-        self.signed = signed
-        self.threshold = threshold
+            if not isinstance(threshold, np.ndarray):
+                threshold = np.asarray(threshold)
```

Comment:
What are the options if the threshold's type is not ndarray? How can you be sure that it can be cast into ndarray?

Reply:
Will be fixed in the following PR that rearrange Keras inferable quantizers